models/experimental/vgg/vgg_utils.py

```diff
@@ -22,7 +22,7 @@ def format_tensor(x, target_layout, device, output_mem_config, pad_value=0.0):
                 x, device, x_padded_shape, pad_value, target_layout, output_mem_config
             )
         else:
-            return tt_lib.tensor.tilize(x, output_mem_config, use_multicore=True)
+            return ttnn.tilize(x, output_mem_config, use_multicore=True)
```

Comment:
For consistency with other ops, memory_config should be a KW argument.

```suggestion
            return ttnn.tilize(x, memory_config=output_mem_config, use_multicore=True)
```

Reply:
Same for `tilize_with_zero_padding`